tests/test_contrib.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2016-2020 CERN.
+#
+# Marshmallow-Utils is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Test the marshmallow contrib."""
+
+import pytest
+from marshmallow import Schema, ValidationError, fields
+
+from marshmallow_utils.fields import contrib
+
+
+def serialize_foo(obj, context):
+    return {'serialize_args': {'obj': 1, 'context': 2}}
+
+
+def deserialize_foo(value, context, data):
+    return {'deserialize_args': {
+        'value': 3, 'context': 4, 'data': 5}}
+
+
+class MySchema(Schema):
+    f = contrib.Function(serialize_foo, deserialize_foo)
+
+
+assert MySchema().dump({'f': 12123}) == {'f': {'serialize_args':
```

Comment:
moderate: could you add this into functions that specify in the name the "reason" of the test? e.g. `test_contrib_function_field_dump`. Same applies for the rest of asserts mainly

Reply:
I made the changes! Thank you!